simpleLooping/binaryListToString.py

```diff
@@ -15,3 +15,15 @@
 # Ex:
 #   binaryListToString([1, 0, 1, 1])       # return "1101"
 #   binaryListToString([0, 1, 1, 0, 0, 1]) # return "100110"
+from typing import List
+
+def binaryListToString(lst: List[int]) -> str:
+    for num in lst:
+        if num != 0 and num != 1:
+            return -1
+
+    binaryStr = ""
+    for i in range(len(lst)):
+        binaryStr += '1' if lst[i] == 0 else '0'
```

Comment:
Are you sure? 

1/ look like lst[i] == 0 will return ‘1’ :))
2/ look like we append this one in the wrong direction?

Ex:
if lst is [0, 1, 0, 1] then your code will return “0101” instead of “1010”

May be “+=“ is not a good idea. Sorry.